connectors/slack/src/main/java/io/camunda/connector/slack/inbound/SlackInboundWebhookExecutable.java

```diff
@@ -89,7 +82,21 @@ public void activate(InboundConnectorContext context) throws Exception {
     if (context == null) {
       throw new Exception("Inbound connector context cannot be null");
     }
-    props = new SlackWebhookProperties(context.getProperties());
+    var wrapperProps = context.bindProperties(SlackConnectorPropertiesWrapper.class);
+    props = new SlackWebhookProperties(wrapperProps);
+  }
+
+  @Override
+  public WebhookHttpVerificationResult verify(WebhookProcessingPayload payload) throws Exception {
+    verifySlackRequestAuthentic(payload);
+    return props
+        .verificationExpression()
+        .apply(
+            Map.of(
+                "body",
+                bodyAsMap(payload.headers(), payload.rawBody()),
```

Comment:
Why do we add the headers to the body? What about query params?

Reply:
We don't. The `bodyAsMap` requires headers to determine content type, because it can be either JSON or form data.
Good point on query params. Keep in mind, the PR is draft and still WIP (work in progress).